contrib/runners/python_runner/python_runner/runner.yaml

```diff
@@ -4,6 +4,25 @@
   enabled: true
   name: python-script
   runner_module: python_runner
+  output_schema:
+    result:
+      anyOf:
+        - type: "object"
```

Comment:
You don't really need to double or single quote strings in YAML.

Reply:
I do for `null` and I'd prefer it be consistent.

Reply:
Another small nit pick - I'm not a fan of our random ordering of attributes in runner.yaml files. It's all over the place. I believe that happened when we converted json to yaml automatically and didn't sort the ordering.